app/views/api/v2/hosts/main.json.rabl

```diff
@@ -40,6 +40,18 @@ node :hostgroup_title do |host|
   host.hostgroup.title if host.hostgroup.present?
 end
 
+if params[:action] == 'show' || (params[:include].present? && params[:include].include?('all_parameters'))
```

Comment:
I think you meant 'parameters' here instead of 'all_parameters', otherwise  you're showing both keys with `all_parameters`